tests/test_mdlp.py

```diff
@@ -32,165 +34,52 @@ def test_find_cut_no_cut():
     k = find_cut(y, 0, len(y))
     assert_equal(-1, k)
 
-def test_mdlp_iris():
-    from sklearn.datasets import load_iris
+def test_e2e():
+  X = np.array([
```

Comment:
You actually don't need `np.array(...)`, because `fit_transform` will transform the input into a `np.array`.

Reply:
This is empirically false.

Reply:
What do you mean?

And if it's not the case, then we have a bug, and we should look into that.

Reply:
Sorry I misspoke I meant TransformerMixin.fit_transform of course 😝

Reply:
See #30 . My mistake!